contracts/zero-ex/contracts/src/features/multiplex/MultiplexUniswapV2.sol

```diff
@@ -143,8 +143,6 @@ abstract contract MultiplexUniswapV2 is
             // Increment the sold and bought amounts.
             state.soldAmount = state.soldAmount.safeAdd(sellAmount);
             state.boughtAmount = state.boughtAmount.safeAdd(boughtAmount);
-        } else {
```

Comment:
why the deletion?

Reply:
I had this for debugging and forgot to take it out 🙊